msg/SensorProperties.msg

```diff
@@ -0,0 +1,7 @@
+string serial                  # Manufacturer specific serial number 
+string sensor_type             # Manufacturer specific name, should be
+                               # used as "<manufacturer>:<product_name>" 
+
+float32 phase_center           # The focal point of the radar waves (radius)
+float32[3] position_resolution # 3 dB resolution in order (radial, azimuth, elevation)
```

Comment:
Perhaps covariance are revelent?

Reply:
( I was working with @tobiasneumann on the specification of this pull request.)

Covariance and resolution are destinct:

| ￬ | Resolution | Covariance |
| --- | --- | --- |
| describes | closest distance between two returns to tell them apart | deviation between the true position and the measured position |
| position error caused by | interference | noise |
| influenced by | bandwidth, antenna characteristic | oscillator stability, signal intensity |

In our modelling the variance would be a property of RadarReturn.
But even if you don't save the upper triangular covariance (like in RadarTrack) and only the diagonal 3 element in spherical coordinates, it would significantly increase the size.

Reply:
From offline conversation: we should not add a breaking change to an existing ROS release, only to rolling

Reply:
I would see this as resolved with the reply from @MRQ.
Can I close this discussion?